src/main.cpp

```diff
@@ -126,6 +126,52 @@ bool is_root()
     return true;
 }
 
+static int info()
+{
+  struct utsname utsname;
+  uname(&utsname);
```

Comment:
Check error code?

Reply:
Don't think that is needed, the only error in `uname(2)` is

> ERRORS
>       EFAULT buf is not valid.

As this is a stack var I don't see how EFAULT could happen